sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/http/client/HttpClient.java

```diff
@@ -20,11 +24,41 @@ public interface HttpClient {
     Response<?> send(HttpRequest request);
 
     /**
-     * Creates a new {@link HttpClient} instance.
+     * Get an instance of the default {@link HttpClient} implementation.
      *
-     * @return A new {@link HttpClient} instance.
+     * @return A new {@link HttpClient} instance, entirely unrelated to all other instances that were created previously
```

Comment:
Don't think the docs here matches the behavior.

Reply:
Agreed - this needs updating to reflect the reality.